hydrocarbon_problem/env/env.py

```diff
@@ -1,13 +1,17 @@
-from typing import List
+from typing import List, Tuple, Optional
 import dm_env
+from dm_env import specs
```

Comment:
What does the dm_env package actually do in our work? I see that we use a couple of things from it (Timestep, specs)
Online I saw that it provides an interface for RL environments

Reply:
Yeah I am following their interface (my environment inherits dm_env.Environment) - similarly to how we have a base class for the aspen api (except here I'm relying on an external library).